packages/block-editor/src/components/block-alignment-control/use-available-alignments.js

```diff
@@ -30,8 +30,9 @@ export default function useAvailableAlignments( controls = DEFAULT_CONTROLS ) {
 	const layoutAlignments = layoutType.getAlignments( layout );
 
 	if ( themeSupportsLayout ) {
-		return layoutAlignments.filter( ( control ) =>
-			controls.includes( control )
+		return layoutAlignments.filter(
+			( { name: alignmentName } ) =>
+				alignmentName === 'none' || controls.includes( alignmentName )
```

Comment:
Instead of filtering `none` here, can't we instead add it to `DEFAULT_CONTROLS` and all the places where it's missing?

Reply:
Addressed in https://github.com/WordPress/gutenberg/pull/34710/commits/292ead1e3d9ce3646056c34a42ea1f937ff72c3d